pkg/report/report.go

```diff
@@ -235,6 +234,13 @@ func (ts *Testsuites) NewSuite(name string) *Testsuite {
 	return suite
 }
 
+// AddFailure adds a failure to the TestSuites collection for startup failures in the test harness
+func (ts *Testsuites) AddFailure(message string) {
```

Comment:
`SetFailure` might be better since it's not a list

Reply:
nice! thx